server/engine/engine.py

```diff
@@ -602,7 +609,8 @@ def processLocalSubjectFolder(path: str, outputName: str = None, href: str = '')
                     # Specifically optimize to 0-ish residuals, if user requests it
                     if residualsToZero:
                         for trial in range(len(dynamicsInit.poseTrials)):
-                            originalTrajectory = dynamicsInit.poseTrials[trial].copy()
+                            originalTrajectory = dynamicsInit.poseTrials[trial].copy(
+                            )
```

Comment:
This is probably auto-formatting, but looks weird to me.

Reply:
Yes, I can't figure out how to convince my auto-formatter to suck less... :/ I'll go fix this manually.